workshops/twine/README.md

```diff
@@ -0,0 +1,141 @@
+# Text Adventure 1: Twine
+
+Welcome! Over the course of the next 3 workshops, you'll be creating your very own text adventure. You'll be starting here, by using a text adventure framework called [Twine](http://twinery.org). It uses an easy-to-follow interface to lay out your story, but you'll also be using some very basic functions called "macros," which we'll be discussing later. [Here's an example of what you'll have created by the end of this workshop.](https://bruggie.com/twine)
```

Comment:
Should we adjust this to remove the "over the course of the next 3 workshops" part?

Also, let's clarify what "functions" means. Maybe use a different word that a beginner may have more luck with?